readme.md

```diff
@@ -230,6 +263,6 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+andy = Instructor.find({name: "Andy"});
```

Comment:
This looks like Active Record Syntax, in Mongoose we can accomplish this behavior via something like:

```
Instructor.findOne({name: "Andy"), function(err, docs){
  docs.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  docs.save();
});
```